examples/CMakeLists.txt

```diff
@@ -23,6 +23,9 @@ endif()
 add_executable("cached-image" WIN32 "cached-image.c" "cached-image.rc")
 target_link_libraries("cached-image" "windrawlib")
 
+add_executable("draw-brush" WIN32 "draw-brush.c")
+target_link_libraries("draw-brush" "windrawlib")
+
```

Comment:
I would rename to `draw-gradient`. The example doesn't do anything else, and likely will never do.